lizmap/modules/lizmap/classes/qgisForm.class.php

```diff
@@ -167,6 +169,35 @@ public function __construct($layer, $form, $featureId, $loginFilteredOverride)
         }
     }
 
+    /**
+     * Get the default value of a QGIS field, if this is
+     * a simple raw value
+     *
+     * @param string $fieldName
+     * @return string|null return null if this is not a number or a string
+     */
+    protected function getDefaultValue($fieldName) {
+        $expression = $this->layer->getDefaultValue($fieldName);
+        if ($expression === null) {
+            return null;
+        }
+        if (is_numeric($expression)) {
+            return $expression;
+        }
+        else if (preg_match("/^'.*'$/", $expression)) {
+            // it seems this is a simple string
+            $expression = trim($expression, "'");
+            // if there are some ' without a \, then probably we have a
+            // true expression, not a single string.
```

Comment:
An other characters for true expression is `"` which is used to decorate field name.